internal/export/dashboards.go

```diff
@@ -51,6 +61,23 @@ func Dashboards(kibanaClient *kibana.Client, dashboardsIDs []string) error {
 	return nil
 }
 
+func checkKibanaVersion(info kibana.VersionInfo) error {
+	version, err := semver.NewVersion(info.Number)
+	if err != nil {
+		return fmt.Errorf("cannot parse version %s: %w", info.Number, err)
+	}
+
+	minVersion := semver.MustParse("8.8.0")
+	maxVersion := semver.MustParse("8.10.0")
```

Comment:
This depends on what value contains `version` variable.

If this variable contains prerelease tags, `8.10.0-SNAPSHOT` would not be allowed to export dashboards:

https://go.dev/play/p/HZD1ehU46FI

Reply:
Tested with 8.10.0-SNAPSHOT and it works as expected.